archaius2-core/src/main/java/com/netflix/archaius/config/polling/FixedPollingStrategy.java

```diff
@@ -29,32 +29,28 @@
 
 public class FixedPollingStrategy implements PollingStrategy {
     private static final Logger LOG = LoggerFactory.getLogger(FixedPollingStrategy.class);
-    
     private final ScheduledExecutorService executor;
     private final long interval;
     private final TimeUnit units;
-    private final boolean syncInit;
-    
+
+
     public FixedPollingStrategy(long interval, TimeUnit units) {
-        this(interval, units, true);
-    }
-    
-    public FixedPollingStrategy(long interval, TimeUnit units, boolean syncInit) {
```

Comment:
Since this is a breaking change let's deprecate this constructor instead and log a warning if syntInit is false.